extensions/notebook/src/book/bookTreeView.ts

```diff
@@ -426,6 +427,7 @@ export class BookTreeViewProvider implements vscode.TreeDataProvider<BookTreeIte
 	public async loadNotebooksInFolder(folderPath: string, urlToOpen?: string, showPreview?: boolean) {
 		let bookCollection = await this.getNotebooksInTree(folderPath);
 		for (let i = 0; i < bookCollection.bookPaths.length; i++) {
+			this.bookQuantity = bookCollection.bookPaths.length;
```

Comment:
>this.bookQuantity = bookCollection.bookPaths.length [](start = 3, length = 51)

optionally you use the i here to expand the books until the setting number and collapse the rest. 

Talk to Julie, if she is fine with the way it is. :thumbsup: